### PR TITLE
fix (countdown-block): refactored initialization to prevent DOM query selector issues and changed desription.

### PR DESCRIPTION
**Pull request by prconcepcion**
Fixes issues with Countdown Block, Fixes https://github.com/gambitph/Stackable/issues/2585

### CODE REVIEW

**Comment by prconcepcion** [comment]
Fixes https://github.com/gambitph/Stackable/issues/2585

File: src/block/countdown/block.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"apiVersion": 2,
 	"name": "stackable/countdown",
 	"title": "Countdown",
-	"description": "Start with the building block of all page layouts.",
+	"description": "Display a countdown timer on your website. This is a great tool for promoting limited-time offers, creating urgency for events, and more.",
```
Comment (bfintal) [inline src/block/countdown/block.json:5]:
```suggestion
	"description": "Display a countdown timer on your website.",
```

File: src/block/countdown/divider.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /**
  * Internal dependencies
  */
+import { i18n } from 'stackable'
```
Comment (bfintal) [inline src/block/countdown/divider.js:4]: Please move this to the external dependencies list.

**Review by bfintal — CHANGES_REQUESTED** [review]
Just one minor change. After adjusting it, please move this to QA review

**Comment by andeng1106** [comment]
Issue (for confirmation):

Some text are not yet translated
- Style > General - this hasn't been translated yet
image
image

- Style > Labels - other text that has been translated before is no longer being translated
    - Hours & Minutes is no longer translated in the screenshot
image
    - Compared to DAnn2012's screenshot: 
image

@prconcepcion @bfintal

**Pull request merged**